gst/nnstreamer/include/nnstreamer_plugin_api.h

```diff
@@ -351,6 +351,23 @@ gst_tensor_get_type (const gchar * typestr);
 extern const gchar *
 gst_tensor_get_type_string (tensor_type type);
 
+/**
+ * @brief Get tensors caps from tensors config and peer caps of the src pad
+ * @param config tensors config structure
+ * @param pad GstPad to check if it supports other/tensor
+ * @return caps for given config and pad
+ */
+extern GstCaps *
+gst_tensors_get_caps (const GstTensorsConfig * config, GstPad * pad);
+
+/**
+ * @brief Check whether pad's caps support other/tensor or not
+ * @param pad GstPad to check if it supports other/tensor
+ * @return TRUE if other/tensor, FALSE if not
+ */
+extern gboolean
+gst_tensor_peer_has_tensor_caps (GstPad * pad);
```

Comment:
How about `gst_pad_peer_has_tensor_caps`?

Reply:
Changed to 'gst_pad_has_tensor_caps'.
Thamks